test-util/src/main/java/io/zeebe/test/util/collection/Maps.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.test.util.collection;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Maps {
+  public static <K, V> Map<K, V> of(Map.Entry<K, V>... entries) {
```

Comment:
Would be nice if we could some newer java lang features heh? :sweat_smile:  like https://www.techiedelight.com/initialize-map-java9/

Reply:
:see_no_evil: